gnocchi/tests/functional/gabbits/resource-type.yaml

```diff
@@ -54,12 +55,8 @@ tests:
                   min_length: 5
                   noexist: foo
       status: 400
-      response_strings:
-          # NOTE(sileht): We would prefer to have a better message but voluptuous seems a bit lost when
-          # an Any have many dict with the same key, here "type"
-          # - "Invalid input: extra keys not allowed @ data[u'attributes'][u'foo'][u'noexist']"
-          # - "Invalid input: not a valid value for dictionary value @ data[u'attributes'][u'foo'][u'type']"
-          - "Invalid input:"
+      response_json_paths:
+        $.description.cause: "Invalid input"
```

Comment:
does this still not include detail/reason?

Reply:
:+1:  Actually sileht's comment is still valid.